### DISCUSSION TITLE
Module Not Found Error: No module named 'pycaret.arules' in Power BI

### DISCUSSION THREAD

**Discussion by goporto20** [pycaret/pycaret#4008] [Q&A]
I am trying to apply association rules with pycaret and the arules module, with the following lines of code:

from pycaret.arules import *
dataset = get_rules(data = dataset, transaction_id = 'customer', item_id = 'product')

But I get the following error: ModuleNotFoundError: No module named 'pycaret.arules'

I was reviewing the documentation to verify if I was doing something incorrectly. But I notice that this module doesn't even appear in the documentation.

Was this module removed? or was it renamed?

Or am I doing something wrong?

**Comment by celestinoxp** [comment]
Hello, you were probably using version 2 and now you are on version 3... a lot has changed since version 2, so I suggest you check the documentation for version 3...  https://pycaret.readthedocs.io/en/latest/index.html

**Reply by goporto20** [reply]
Thanks for your comments, I'll check it out.

I was actually complementing pycaret with a bi tool and that's where I noticed those differences.

Very nice, thank you very much
